dvc_ssh/__init__.py

```diff
@@ -105,14 +85,8 @@ def _prepare_credentials(self, **config):
             "aes128-ctr",
         ]
         login_info["compression_algs"] = None
-
         login_info["gss_auth"] = config.get("gss_auth", False)
         login_info["agent_forwarding"] = config.get("agent_forwarding", True)
-        login_info["proxy_command"] = user_ssh_config.get("ProxyCommand")
-
-        # We are going to automatically add stuff to known_hosts
-        # something like paramiko's AutoAddPolicy()
-        login_info["known_hosts"] = None
```

Comment:
This is already set in `sshfs`.

https://github.com/fsspec/sshfs/blob/47d78ac22daafc6c2aa2dd8de678f1413ef0dadd/sshfs/spec.py#L63